src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+enum Colors {
```

Comment:
It should also be public + enum names should be in singular form:

```suggestion
public enum Color {
```